### PR TITLE
Add links to video container

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
## What does this change?

It adds links to the new video container. I've had to do some weird stuff with `pointer-events` and `z-index` to work as we want the rest of the element to play the video.
## What is the value of this and can you measure success?

We'll get some click throughs. If that's your bag
## Screenshots

screen recording 2016-04-21 at 02 31 pm
## Request for comment

@sndrs @jamesgorrie

### CODE REVIEW

File: static/src/stylesheets/module/facia/_container--video.scss [hunk 1]
```diff
@@ -338,6 +342,14 @@ $video-width-desktop: 700px;
         fill: #ffffff;
     }
 
+    .fc-item__link {
+        color: #ffffff;
+
+        &:visited {
+            color: $neutral-3
```
Comment (jamesgorrie) [inline static/src/stylesheets/module/facia/_container--video.scss]: +;?

File: static/src/stylesheets/module/facia/_container--video.scss [hunk 2]
```diff
@@ -43,6 +43,8 @@ $video-width-desktop: 700px;
         left: 0;
         right: 0;
         bottom: 0;
+        pointer-events: none;
```
Comment (jamesgorrie) [inline static/src/stylesheets/module/facia/_container--video.scss]: have we made sure this isn't affecting any other containers (or is `.fc-container--video` new for this container?)

Comment (jamesgorrie) [inline static/src/stylesheets/module/facia/_container--video.scss]: Also - we should at least try something different as this get's rid of selection etc.

Comment (sammorrisdesign) [inline static/src/stylesheets/module/facia/_container--video.scss]: This is scoped to the `.video-overlay` element, so no need there.

Also selection is essentially disabled on all our items on fronts, due to the `faux-link` overlay.

Comment (jamesgorrie) [inline static/src/stylesheets/module/facia/_container--video.scss]: > faux-link overlay.

Oh yes 🙍

**Comment by jamesgorrie** [comment]
👍

**Comment by paperboyo** [comment]
Nice, thanks!

**Pull request merged**